hedyorg/hedy
# 💻 5959 shorten error messages

Fixes #5959 
Execute error and other error (ServerError) are shortened. 

**How to test**

Follow these steps to verify this PR works as intended:

```repeat 100 times print 'Hello Hedy'```
* go to level 7
* run ```repeat 100 times print 'Hello Hedy'```
* the error messages will be: 'Something went wrong. Your program takes too long to run.'
* @jpelay I don't know how to get the other_error/ServerError, to test it.

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)
